src/components/projects/projectId/lookup-lists/lookupListId/Terms.tsx

```diff
@@ -1,11 +1,10 @@
 import { selectProjectId } from "@/src/reduxStore/states/project";
-import { ADD_TERM_TO_LOOKUP_LIST, BLACKLIST_TERM, REMOVE_TERM, UPDATE_TERM } from "@/src/services/gql/mutations/lookup-lists";
+import { deleteTerm as dlt, blacklistTerm as blt, addTermToKnowledgeBase as tkb, updateTerm } from "@/src/services/base/lookup-lists";
```

Comment:
i think the naming is not clear, if i take a look at the code in few weeks i wouldn't know what's blt and tkb. The names aren't that long so i would go with a full name